### DISCUSSION TITLE
Vulnerability discovered by IIIT Hyderabad Students

### DISCUSSION THREAD

**Discussion by ghost** [Kunzisoft/KeePassDX#1758] [Q&A] [answered]
Given that Keepass2Android is similar to Keepassdx, is Keepassdx safe from this vulnerability?
Link to article: https://indianexpress.com/article/technology/tech-news-technology/iiit-hyderabad-autospill-attack-android-password-managers-9064721/

**Comment by shuvashish76** [comment, marked as answer]
Known issue. Duplicate of #1716
